extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -308,6 +326,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   /** @override */
   buildCallback() {
     super.buildCallback();
+    this.isFluid_ = this.element.getAttribute('height') == 'fluid';
+    this.creativeSize = this.isFluid_ ?
```

Comment:
Ideally we wouldn't need to touch creativeSize here so it could remain protected.   Let's discuss offline.

Reply:
Done.